apps/bbsync/tests/test_cc.py

```diff
@@ -0,0 +1,809 @@
+        """
+        test that notaffected affects result in empty CCs
+        however this restriction only applies to embargoed flaws
+        """
+        flaw = FlawFactory(embargoed=embargoed)
+        affect1 = AffectFactory(
+            flaw=flaw, affectedness=Affect.AffectAffectedness.NOTAFFECTED
+        )
+        affect2 = AffectFactory(
+            flaw=flaw,
+            affectedness=Affect.AffectAffectedness.AFFECTED,
+            resolution=Affect.AffectResolution.WONTFIX,
+        )
+        PsModuleFactory(
+            name=affect1.ps_module,
+            default_cc=["me@redhat.com"],
+        )
+        PsModuleFactory(
+            name=affect2.ps_module,
+            default_cc=["you@redhat.com"],
+        )
+
+        cc_builder = CCBuilder(flaw)
+        add_cc, remove_cc = cc_builder.content
+        assert add_cc == cc
+        assert not remove_cc
+
+    ###############
+    # FLAW UPDATE #
+    ###############
+
+    def test_no_change(self):
+        """
+        test that no change means no change
+        """
+        flaw = self.prepare_flaw(
+            affects=[
+                ("rhel-6", "kernel"),
+                ("rhel-7", "openssl"),
+            ],
+            old_affects=[
+                ("rhel-6", "kernel"),
+                ("rhel-7", "openssl"),
+            ],
+            old_cc=[
+                "rhel-6.kernel@redhat.com",
+                "rhel-7.openssl@redhat.com",
+            ],
+        )
+
+        cc_builder = CCBuilder(flaw, old_flaw=flaw)
+        add_cc, remove_cc = cc_builder.content
+        assert not add_cc
+        assert not remove_cc
+
+    def test_no_change_removed(self):
+        """
+        test that no change means no change
+        even when someone has been manually removed
+        """
+        flaw = self.prepare_flaw(
+            affects=[
+                ("rhel-6", "kernel"),
+                ("rhel-7", "openssl"),
+            ],
+            old_affects=[
+                ("rhel-6", "kernel"),
+                ("rhel-7", "openssl"),
+            ],
+            old_cc=[
+                "rhel-6.kernel@redhat.com",
+                # rhel-7 CC is missing here
+            ],
+        )
+
+        cc_builder = CCBuilder(flaw, old_flaw=flaw)
+        add_cc, remove_cc = cc_builder.content
+        assert not add_cc
+        assert not remove_cc
+
+    def test_no_change_extra(self):
+        """
+        test that no change means no change
+        even when someone has been manually added
+        """
+        flaw = self.prepare_flaw(
+            affects=[
+                ("rhel-6", "kernel"),
+                ("rhel-7", "openssl"),
+            ],
+            old_affects=[
+                ("rhel-6", "kernel"),
+                ("rhel-7", "openssl"),
+            ],
+            old_cc=[
+                "rhel-6.kernel@redhat.com",
+                "rhel-7.openssl@redhat.com",
+                "random.user@email.com",
```

Comment:
what happens when there has been a change and someone has been manually added/removed?